File: SoalA/C++/DISK.cpp
#include <iostream>
#include <string>

using namespace std;

class DISK
{
    /* membuat atribut */
private:
    int priceDISK;
    string typeDISK;
    int capacityDISK;

public:
    /* constructor */
    DISK(int priceDISK, string typeDISK, int capacityDISK)
    {
        this->priceDISK = priceDISK;
        this->typeDISK = typeDISK;
        this->capacityDISK = capacityDISK;
    }

    DISK()
    {
    }

    /* Membuat get set method harga DISK */
    void setPriceDISK(int priceDISK)
    {
        this->priceDISK = priceDISK;
    }
    int getPriceDISK()
    {
        return this->priceDISK;
    }

    /* Membuat get set method tipe DISK */
    void setTypeDISK(string typeDISK)
    {
        this->typeDISK = typeDISK;
    }
    string getTypeDISK()
    {
        return this->typeDISK;
    }

    /* Membuat get set method kapasitas DISK */
    void setCapacityDISK(int capacityDISK)
    {
        this->capacityDISK = capacityDISK;
    }
    int getCapacityDISK()
    {
        return this->capacityDISK;
    }

    /* menampilkan data DISK */
    void printDISK()
    {
        cout << "DISK Price(IDR)    : " << this->getPriceDISK() << endl;
        cout << "DISK Type          : " << this->getTypeDISK() << endl;
        cout << "DISK Capacity(GB)  : " << this->getCapacityDISK() << endl;
        cout << "====================================================" << endl;
    }

    /* destructor */
    ~DISK()
    {
    }
};

File: SoalB/Pyhton/DISK.py
class DISK():
    # atribut
    __priceDISK = 0
    __typeDISK = ""
    __capacityDISK = 0

    # constructor
    def __init__(self):
        self.__priceDISK = 0
        self.__typeDISK = ""
        self.__capacityDISK = 0

    # membuat get set method harga DISK
    def setPriceDISK(self, priceDISK):
        self.__priceDISK = priceDISK

    def getPriceDISK(self):
        return self.__priceDISK

    # membuat get set method tipe DISK
    def setTypeDISK(self, typeDISK):
        self.__typeDISK = typeDISK

    def getTypeDISK(self):
        return self.__typeDISK

    # membuat get set method kapasitas disk
    def setCapacityDISK(self, capacityDISK):
        self.__capacityDISK = capacityDISK

    def getCapacityDISK(self):
        return self.__capacityDISK

    # menampilkan data disk
    def printDISK(self):
        print("DISK Price(IDR)      : " + str(self.getPriceDISK()))
        print("DISK Type            : " + str(self.getTypeDISK()))
        print("DISK Capacity(GB)    : " + str(self.getCapacityDISK()))
        print("---------------------------------------")
